docs/docusaurus/docs/conceptual_guides/gx_overview.md

```diff
@@ -53,17 +53,17 @@ You can configure where your Data Docs are hosted.  Unlike Stores, you can defin
 
 For more information, see [Host and share Data Docs](/docs/guides/setup/configuring_data_docs/host_and_share_data_docs).
 
-## Datasources
+## Data Sources
 
 ![Data Source overview](./images_overview/datasource_flowchart.png)
 
-Datasources connect GX to source data such as CSV files in a folder, a PostgreSQL database hosted on AWS, or any combination of data formats and environments. Regardless of the format of your source data or where it resides, Datasources provide GX with a unified API for working with it.
+Data Sources connect GX to Data Assets such as CSV files in a folder, a PostgreSQL database hosted on AWS, or any combination of data formats and environments. Regardless of the format of your Data Asset or where it resides, Data Sources provide GX with a unified API for working with it.
 
 For more information, see [Connect to data](/docs/guides/connecting_to_your_data/connect_to_data_lp).
 
 ### Data Assets and Batches
 
-Data Assets are collections of records within a Data Source.  While a Data Source tells GX how to connect to your source data, Data Assets tell GX how to organize that data. Although the records in your Data Assets can correspond directly to the contents of tables or files in your source data they do not necessarily need to. For instance, you could combine multiple tables worth of records in a SQL Data Source into a single Query Data Asset that joins the tables in question.  For a File Data Source, you could use regular expressions to define a Data Asset as the contents of all of the `.csv` files in a specific subfolder.
+Data Assets are collections of records within a Data Source.  A Data Source tells GX how to connect to your Data Assets and Data Assets tell GX how to organize that data. Although the records in your Data Assets can correspond directly to the contents of tables or files in your Data Source they do not necessarily need to. For instance, you could combine multiple tables worth of records in a SQL Data Source into a single Query Data Asset that joins the tables in question.  For a File Data Source, you could use regular expressions to define a Data Asset as the contents of all the `.csv` files in a specific subfolder.
```

Comment:
I noticed that we're pretty inconsistent with single vs double spaces after periods. It looks like it doesn't affect how the docs are ultimately rendered, but it's perhaps something worth keeping an eye on.

Reply:
@r34ctor I've noticed this issue, too - the joys of legacy docs. Ideally, single spacing should be used.